public/templates/profile-completion-message.html

```diff
@@ -19,4 +19,11 @@
       </div>
     </div>
   </div>
-</div>
\ No newline at end of file
+</div>
+<div class="container profile-completion-message" ng-if="user && user.id && user.email && !user.email_verified">
+  <div class="panel panel-warning">
+    <div class="panel-heading">
+      <p>Uh oh, we noticed you haven't verified your email address yet. We need you to verify it before we can send messages to it. Please check your inbox or <a href="" ng-click="resendEmailVerification()" >resend the verification email</a>.</p>
```

Comment:
If you could, this needs to be localized like the other templates.  It looks like I missed it in my first pass.